infra/build/functions/build_and_push_test_images.py

```diff
@@ -57,16 +64,46 @@ def build_image(image, tags):
   logging.info('Built: %s', image)
 
 
-def build_and_push_images(test_image_suffix):
+def gcb_build_and_push_images(test_image_suffix):
+  """Build and push test versions of base images using GCB."""
+  steps = [build_lib.get_git_clone_step()]
+  test_images = []
+  for base_image in base_images.BASE_IMAGES:
+    image_name = TAG_PREFIX + base_image
+    test_image_name = f'{image_name}-{test_image_suffix}'
+    test_images.append(test_image_name)
+    directory = os.path.join('infra', 'base-images', base_image)
+    step = build_lib.get_docker_build_step([image_name, test_image_name],
+                                           directory)
+    steps.append(step)
+
+  overrides = {'images': test_images}
+  credentials = oauth2client.client.GoogleCredentials.get_application_default()
+  build_id = build_lib.run_build(steps, credentials, base_images.BASE_PROJECT,
+                                 base_images.TIMEOUT, overrides,
+                                 ['trial-build'])
+  return trial_build.wait_on_builds({'base-images': build_id}, credentials,
+                                    CLOUD_PROJECT)
+
+
+def build_and_push_images(test_image_suffix, local=True):
```

Comment:
Rather than this default arg and overloading this function, can we just have a separate `local_build_and_push_images` to make things more explicit?